types/flexslider/flexslider-tests.ts

```diff
@@ -1,21 +1,17 @@
-import $ = require('jquery');
-
-// Can also be used with $(document).ready()
-$(window).load(function() {
+$(window).ready(function() {
```

Comment:
This is bad code though 😆 

https://jquery.com/upgrade-guide/3.0/#deprecated-document-ready-handlers-other-than-jquery-function